packages/toast/src/ToastContainer.tsx

```diff
@@ -0,0 +1,63 @@
+import { ToastContainer as SourceToastContainer } from 'react-toastify'
+import { injectStyle } from 'react-toastify/dist/inject-style'
```

Comment:
The build crashes on this line

Reply:
so, i passed tests, but had to use commonjs.namedExports (deprecated), more details:
...
import commonjs from '@rollup/plugin-commonjs'
...
plugins: [
        ...
        commonjs({
          namedExports: {
            './.yarn/react-toastify/dist/inject-style.js': ['injectStyle'],
          },
        }),
]
...

Reply:
Official documentation gave broken link to https://github.com/rollup/plugins/tree/master/packages/commonjs#custom-named-exports (anchor 'custom-named-exports' does not exist)...

So, I have hit a wall :)

Reply:
@avsetsin, do you know the solution?

Reply:
I don't really like this `inject-style` solution for our library. We use styled-components, and another way to add styles makes the build and the support more complicated. For example, someday we will want to cover the library with tests and this package will require extra effort because of the different way of styling.

I think it would be better to copy the default toastify styles to `ToastifyContainer` and edit the css to match our design. Or even write everything from scratch: https://fkhadra.github.io/react-toastify/how-to-style/#how-to-style-with-styled-components. It will help us to keep the same injection of styles in all packages, and also we can get rid of some of the default styles that we do not need